client/public/index.html

```diff
@@ -9,6 +9,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600&display=swap" rel="stylesheet">
```

Comment:
Maybe we should use .tff files for fonts. But it wont make its performance better, so let it be! 😛

Reply:
Yes, that might not make any difference in performance as same specified font files are loaded in both the cases and google font api provide all specified font files in compressed form for faster download.I might be wrong on this. 
So should we need to add .ttf/.otf files for fonts or leave it like this?

Reply:
Nope. I dont think we should be investing time in setting up ttf without having any obvious performance boost.